src/overlays/actors/ovl_Dm_Char08/z_dm_char08.h

```diff
@@ -8,11 +8,37 @@ struct DmChar08;
 typedef void (*DmChar08ActionFunc)(struct DmChar08*, GlobalContext*);
 
 typedef struct DmChar08 {
-    /* 0x0000 */ Actor actor;
-    /* 0x0144 */ char unk_144[0x5C];
-    /* 0x01A0 */ DmChar08ActionFunc actionFunc;
-    /* 0x01A4 */ char unk_1A4[0x68];
-} DmChar08; // size = 0x20C
+    /* 0x000 */ DynaPolyActor dyna;
+    /* 0x15C */ SkelAnime skelAnime;
+    /* 0x1A0 */ DmChar08ActionFunc actionFunc;
+    /* 0x1A4 */ Actor* tree1;
+    /* 0x1A8 */ Actor* tree2;
+    /* 0x1AC */ Vec3f tree1Pos;
+    /* 0x1B8 */ Vec3f tree2Pos;
+    /* 0x1C4 */ Vec3f bubblePos;
+    /* 0x1D0 */ Vec3f focusPos;
+    /* 0x1DC */ char pad1DC[8];
```

Comment:
```suggestion
    /* 0x1DC */ UNK_TYPE1 pad1DC[8];
```
Same for the others.